File: app/src/main/java/org/deiverbum/app/model/IntroViaCrucis.java
package org.deiverbum.app.model;

import org.deiverbum.app.utils.Utils;

public class IntroViaCrucis {
    String saludo;
    String intro;
    String oracion;

    @SuppressWarnings("unused")
    public String getSaludo() {
        return this.saludo;
    }

    @SuppressWarnings("unused")
    public void setSaludo(String saludo) {
        this.saludo = saludo;
    }

    public String getIntro() {
        return Utils.getFormato(intro);
    }

    public void setIntro(String intro) {
        this.intro = intro;
    }

    public String getOracion() {
        return Utils.getFormato(oracion);
    }

    public void setOracion(String oracion) {
        this.oracion = oracion;
    }

}

File: app/src/main/java/org/deiverbum/app/core/model/alteri/IntroViaCrucis.kt
package org.deiverbum.app.core.model.alteri

import org.deiverbum.app.util.Utils

class IntroViaCrucis {
    @Suppress("unused")
    var saludo: String? = null
    var intro: String? = null
        get() = if (field?.isNotEmpty() == true) Utils.getFormato(field!!) else ""
    var oracion: String? = null
        get() = if (field?.isNotEmpty() == true) Utils.getFormato(field!!) else ""
}